src/parse.c

```diff
@@ -1909,7 +1909,7 @@ handle_vxlan_tristate(NetplanParser* npp, yaml_node_t* node, const void* data, G
 STATIC int
 get_ip_family(const char* address)
 {
-    g_autofree char *ip_str;
+    g_autofree char *ip_str = NULL;
```

Comment:
The "rpm fedora:latest" check uses -Werror to make warnings abort compilation. This "may be used uninitialized" warning was reported in the log for this check when I initially uploaded and I thought this may be what was preventing the check from completing successfully.

It's a reasonable change as long as it's retained as a separate commit as I have uploaded it. If your policy is to merge pull requests by squashing all of the commits into a single commit, then we should definitely discard this one before merging. Or, even if you don't merge by squashing, this commit can be dropped if you prefer since the "rpm fedora:latest" check is just broken, and hence it's not strictly necessary for the following routing-rule commit.

Reply:
I agree, we can keep it as a separate commit. Thanks for providing additional context!

Reply:
Actually, I'm cherry-picking this commit into https://github.com/canonical/netplan/pull/527 to fix the "rpmbuild" CI